### ISSUE TITLE
Will the training dataset be open-sourced?

### ISSUE THREAD

**Issue by Jacob-Chow** [lcqysl/FrameThinker_open#4]
非常出色的工作！

想问问强化学习部分的parquet能给少量案例么

感谢🙏

**Comment by lcqysl** [comment]
{
    "data_source": "vstar",
    "prompt": [
        {
            "content": "You are an expert AI assistant that answers questions about a video by iteratively analyzing it.\nYour task is to output your reasoning within a <think> </think> tag, followed by a specific action within an <action> </action> tag.\nPossible actions are:\n1. `choose frames between START_FRAME and END_FRAME`: Request a more detailed view of a specific video segment. The number of frames is fixed, currently 8.\n2. `get frame number at time MM:SS`: Get the exact frame number for a specific time. Convert hours to minutes if needed (e.g., for 1 hour, 2 minutes, and 30 seconds, use 62:30).\n3. `output answer: OPTION`: Provide the final answer (e.g., A, B, C...) when you are confident.",
            "role": "system"
        },
        {
            "content": "Could the person have transferred what they are concealing from one hand to the other while shuffling their hands?\nA. What they conceal is transferrable but there was no clear moment when that could have happened.\nB. What they conceal is not transferrable.\nC. What they conceal is transferrable and there were moments when that could have happened.\n\nframe 0: <image>\nframe 64: <image>\nframe 129: <image>\nframe 194: <image>\nframe 258: <image>\nframe 323: <image>\nframe 388: <image>\nframe 453: <image>",
            "role": "user"
        }
    ],
    "images": [
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...tco5N03Ttu3/A+yeAcPS68p2AAAAAElFTkSuQmCC",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000000.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...pmmyzcw2C0n/D7AmftUy6DsJAAAAAElFTkSuQmCC",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000064.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...799fXV2psE3xJ9hxSpcz97Y5AAAAAElFTkSuQmCC",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000129.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...RRJ/GME/JAH8AKh704RNCr3cAAAAAElFTkSuQmCC",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000194.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...TEmAJMA2YPu/GV3qLG914g8AAAAASUVORK5CYII=",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000258.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...fLLoSWLYMMD/D5+sAtv/pkGpAAAAAElFTkSuQmCC",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000323.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...2+12p6en/z/yOMqLMmIXYgAAAABJRU5ErkJggg==",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000388.png",
            "encoding": "base64"
        },
        {
            "bytes": "iVBORw0KGgoAAAANSUhEUgAAAoAAAAFoCAIAAABI... (omitted for brevity) ...QDqgASKLJKz+D26LGWVseNYlAAAAAElFTkSuQmCC",
            "path": "/cpfs01/user/hezefeng/data/Perception/frames/video_4694/frame_000453.png",
            "encoding": "base64"
        }
    ],
    "ability": "vl_video_reasoning",
    "env_name": "think_with_video",
    "reward_model": {
        "ground_truth": "A",
        "style": "rule"
    },
    "extra_info": {
        "answer": "A",
        "fps": 30.019,
        "height": 360,
        "question": "Could the person have transferred what they are concealing from one hand to the other while shuffling their hands?",
        "split": "train",
        "total_frames": 454,
        "video_id": "video_4694",
        "video_path": "/cpfs01/user/hezefeng/data/Perception/videos/video_4694.mp4",
        "width": 640
    }
}
